core/lspserver.py

```diff
@@ -347,7 +347,7 @@ def send_completion_request(self, request_id, filepath, type, row, column, char)
     def send_find_define_request(self, request_id, filepath, type, row, column):
         self.record_request_id(request_id, filepath, type)
 
-        self.send_to_request("textDocument/declaration",
+        self.send_to_request("textDocument/definition",
```

Comment:
Since this method is `send_find_define_request`, it make more sense to send `definition` message.

Reply:
It's ok.

We can add https://microsoft.github.io/language-server-protocol/specification#textDocument_declaration and https://microsoft.github.io/language-server-protocol/specification#textDocument_typeDefinition later.